aspnet/conceptual-overview/aspnet.rst

```diff
@@ -85,17 +107,17 @@ You can use any `OWIN <http://owin.org>`_-based middleware with ASP.NET Core. Se
 Servers
 -------
 
-The ASP.NET Core hosting model does not directly listen for requests, but instead relies on an HTTP :doc:`server </fundamentals/servers>` implementation to surface the request to the application as a set of feature interfaces that can be composed into an HttpContext. ASP.NET Core includes a managed cross-platform web server, called :ref:`Kestrel <kestrel>`, that you would typically run behind a production web server like `IIS <https://iis.net>`__ or `nginx <http://nginx.org>`__.
+The ASP.NET Core hosting model does not directly listen for requests, but instead relies on an HTTP :doc:`server </fundamentals/servers>` implementation to surface the request to the application as a set of feature interfaces that can be composed into an ``HttpContext``. ASP.NET Core includes a managed cross-platform web server, called :ref:`Kestrel <kestrel>`, that you would typically run behind a production web server like `IIS <https://iis.net>`__ or `nginx <http://nginx.org>`__.
 
 Web root
 --------
 
-The Web root of your application is the root location in your project from which HTTP requests are handled (ex. handling of static file requests). The Web root of an ASP.NET Core application is configured using the "webroot" property in your project.json file.
+The web root of your app is the root location in your project from which HTTP requests are handled (for example handling of static file requests). The web root of an ASP.NET Core app is configured using the ``webroot`` property in the *project.json* file.
```

Comment:
> The web root of an ASP.NET Core app is configured using the ``webroot`` property in the *project.json* file.

Remove. This is no longer true.

Instead add: "For example, when using the static files middleware only files from the web root folder will be made accessible. The rest of the files in your content root can not be accessed remotely. the default web root path is `<content root>/wwwroot`, but you can specify a different location using the `WebHostBuilder`.